src/js/components/Calendar/__tests__/Calendar-test.js

```diff
@@ -13,7 +13,7 @@ import { Box, Button, Calendar, Grommet, Text } from '../..';
 const DATE = '2020-01-15T00:00:00-08:00';
 const DATES = [
   '2020-01-12T00:00:00-08:00',
-  ['2020-01-8T00:00:00-08:00', '2020-01-10T00:00:00-08:00'],
+  ['2020-01-08T00:00:00-08:00', '2020-01-10T00:00:00-08:00'],
```

Comment:
Why was this date changed? I think this is causing the snapshot changes

Reply:
Yes.
I'm not sure if I'm supposed to change that, but as I described in the PR, I considered that a typo.
`DATES` is an Array that contains another in the second position. That means that Calendar should render two selections:
1. Single Date (Jan 12)
2. Range Date (Jan 8 - Jan 10)

Value `2020-01-8T00:00:00-08:00` doesn't seem to be recognized as valid, making Calendar render this:
![image](https://user-images.githubusercontent.com/47114840/94963771-03988280-04cf-11eb-991f-6bee4a1d8dd1.png)

instead of:

![image](https://user-images.githubusercontent.com/47114840/94963804-0eebae00-04cf-11eb-8ea6-b2d062c9a235.png)


**Code used is the same as DATES Test**
```
import React from "react";
import { Grommet, Calendar } from "grommet";

const DATES = [
  "2020-01-12T00:00:00-08:00",
  ["2020-01-8T00:00:00-08:00", "2020-01-10T00:00:00-08:00"],
];

const App = () => {
  return (
    <Grommet>
      <Calendar dates={DATES} animate={false} />
    </Grommet>
  );
};

export default App;
```

Reply:
Am I supposed to turn that into a separate PR?

Reply:
Ah okay I see, I think it is fine to leave it in this PR instead of a separate one, what do you think @ShimiSun?

Reply:
I agree we can keep it.